_posts/2021-04-19-security-incident-disclosure.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Security Incident Disclosure
+author: Markus Reiter
+---
+
+On 18th April 2021 a security researcher identified a vulnerability in our `review-cask-pr` GitHub action which is used to automatically review and approve version bump pull requests. This was reported via Hacker One and [a pull request demonstrating the vulnerability](https://github.com/Homebrew/homebrew-cask/pull/104191) was submitted in accordance with us. The [pull request was subsequently reverted](https://github.com/Homebrew/homebrew-cask/pull/104197) and the [`review-cask-pr` action disabled and removed](https://github.com/Homebrew/homebrew-cask/pull/104204) from all repositories that were using it.
+
+### What was impacted
+
+The discovered vulnerability would allow an attacker to inject arbitrary payloads into a cask and have it
+be merged automatically. This is due to a bug in the `git_diff` dependency of the `review-cask-pr` GitHub action which is used to parse a pull request's diff for inspection. Due to this bug, the parser can be spoofed into completely ignoring the offending lines, resulting in successfully approving a malicious pull request.
```

Comment:
```suggestion
be merged automatically. This is due to a flaw in the `git_diff` dependency of the `review-cask-pr` GitHub action which is used to parse a pull request's diff for inspection. Due to this flaw, the parser can be spoofed into completely ignoring the offending lines, resulting in successfully approving a malicious pull request.
```

Part of me feels that "flaw" is a better word to use instead of "bug" here. To me, bug implies something that we're fixing and I don't want to give the impression that we are going to fix the `review-cask-pr` action and continue using it.